modules/apps/dispatch/dispatch-talend-web/src/main/java/com/liferay/dispatch/talend/web/internal/process/TalendProcess.java

```diff
@@ -112,6 +135,19 @@ public Builder companyId(long companyId) {
 		}
 
 		public Builder contextParam(String name, String value) {
+			if (name.startsWith("-D") || name.startsWith("-X")) {
```

Comment:
@shuyangzhou do we have any Util I could use to handle JVM options? User can pass options via UI and I would like to validate it before passing to sub-process.

Reply:
We have no such thing, you have to collect your own jvm args.